dev-tools/info-extractor.php

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env php
```

Comment:
>php-cs-fixer about command (or sth)

how about `php-cs-fixer -V --format=json` ?

Reply:
not sure how nice it would be to modify built-in `--version` pseudocommand

anyway, i want to have more than just version exposed, also available rules, sets and so